lib/CGameInfoCallback.cpp

```diff
@@ -970,17 +971,27 @@ const CGObjectInstance * CGameInfoCallback::getObjInstance( ObjectInstanceID oid
 
 CArtifactSet * CGameInfoCallback::getArtSet(const ArtifactLocation & loc) const
```

Comment:
I do see that this is old code, but such methods should not return non-const entities.

Reply:
> I do see that this is old code, but such methods should not return non-const entities.

How do you propose to get non-constant instances in this case?

Reply:
CGameInfoCallback should only provide const API.

This method probably should be moved to CGameState and made non-const
And CGameState does provides non-const API for object access: `getObjInstance` or `getHero`

Reply:
Moved